README.md

```diff
@@ -69,7 +69,25 @@ jobs:
           container: ubuntu:20.04
 ```
 
+### `branch`
+
+**Optional** Branch passed to FOSSA CLI.
+
+Example
+```yml
+jobs:
+  fossa-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
```suggestion
      - uses: actions/checkout@v3
```
Small nit: Latest checkout action is `v3`. You could also update the runtime to node16 in `action.yml`